frontend/templates/views/partials/component-sidebar.j2

```diff
@@ -32,7 +32,8 @@
                   {% endif %}
                   {% set formats = component.formats or [] %}
                   {% set activeDocument = true if component.url == doc.url %}
-                  [% if format in {{ formats|tojson }} %]
+                  {# the latest document might not support a format, hence we have to check if any version supports the format #}
+                  [% if format in {{ component.supported_formats|list|tojson }} %]
```

Comment:
I think this is the first time we really used the supported_formats property. But we could also use the generated json structure and get rid of the `supported_formats` property in the frontmatter. For nearly all components the values are the same as the normal `formats` anyway. We would not rely on the manually maintained `componentFormats.json` but only on the data we import.

Reply:
You're right. Fixed.

Reply:
The `supported_formats` is still used here.
We should use the `format-component-mapping.json` data instead.